packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -52,11 +47,21 @@ const Color slowFrameColor = Color(0xFFE50C0C);
 // Blue A700 is light, Indigo A400 is dark
 const Color selectedGpuColor =
     ThemedColor(Color(0xFF2962FF), Color(0xFF3D5AFE));
-// Dark Blue is light, Indigo A200 is dark
+// Dark Blue is light, Deep Purple A200 is dark
 const Color selectedCpuColor =
-    ThemedColor(Color(0xFF09007E), Color(0xFF536DFE));
-// Selection is high-contrast need white font.
-const Color hoverTextColor = ThemedColor(Colors.white, defaultForeground);
+    ThemedColor(Color(0xFF09007E), Color(0xFF7C4DFF));
+
+// Text color defaultForeground is too gray hard to read text, make it a little
+// whitter, not jarring white #FFFFFF.
+const ThemedColor contrastForeground =
+    ThemedColor(Colors.black, Color.fromARGB(255, 240, 240, 240));
+
+// Jank/Selection is high-contrast need white-ish font.
+const Color hoverTextHiContrastColor =
+    ThemedColor(Colors.white, contrastForeground);
+// Other hovers are not as contrasty (good frames) black text looks best in both
+// light and dark mode.
+const Color hoverTextColor = ThemedColor(Colors.black, Colors.black);
```

Comment:
Can we just use Colors.black here instead of a ThemedColor?

Reply:
Mostly for consistency.  When a UX designer goes over our code we won't have to change N places in our code just constants in one place.  Hopefully in one file that has all our color items.